README.md

```diff
@@ -1,16 +1,68 @@
-## Google Cloud Platform Python Samples
+# Google Cloud Platform Python Samples
 
-[![Open in Cloud Shell][shell_img]][shell_link]
+Python samples for [Google Cloud Platform products][cloud].
 
-[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
-[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=./README.md
 
-This repository holds the samples used in the python documentation on [cloud.google.com](https://cloud.google.com).
+[![Build Status][py-2.7-shield]][py-2.7-link] [![Build Status][py-3.6-shield]][py-3.6-link] [![Build Status][py-3.7-shield]][py-3.7-link] [![Build Status][py-3.8-shield]][py-3.8-link]
 
-[![Build Status][py-2.7-shield]][py-2.7-link]
-[![Build Status][py-3.6-shield]][py-3.6-link]
-[![Build Status][py-3.7-shield]][py-3.7-link]
-[![Build Status][py-3.8-shield]][py-3.8-link]
+## Setup
+
+1. Install [`pip` and `virtualenv`][cloud_python_setup] if you do not already have them.
+
+1. Clone this repository:
+
+    ```
+    git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+    ```
+
+1. Obtain authentication credentials.
+
+    Create local credentials by running the following command and following the
+    oauth2 flow (read more about the command [here][auth_command]):
+
+    ```
+    gcloud auth application-default login
+    ```
+
+    Read more about [Google Cloud Platform Authentication][gcp_auth].
+
+## How to run a sample
+
+1. Change directory to one of the sample folders, e.g. `logging/cloud-client`:
+
+    ```
+    cd logging/cloud-client/
+    ```
+
+1. Create a virtualenv. Samples are compatible with Python 2.7 and 3.4+.
+
+    ```
+    virtualenv env
+    source env/bin/activate
+    ```
+
+1. Install the dependencies needed to run the samples.
+
+    ```
+    pip install -r requirements.txt
+    ```
+
+1. Run the sample:
+
+    ```
+    python snippets.py
+    ```
+
+## Contributing
+
+Contributions welcome! See the [Contributing Guide][CONTRIBUTING.md].
```

Comment:
Need parens, not square brackets, around CONTRIBUTING.md.

Reply:
Fixed. :)